scripts/benchmark-rule.mjs

```diff
@@ -51,9 +52,43 @@ const processor = postcss().use(rule);
 fetch(CSS_URL)
 	.then((response) => response.text())
 	.then((response) => {
+		let css = '';
+
+		for (let i = 0; i < 20; i++) {
```

Comment:
suggestion

put the magic number in a variable and maybe add a comment to explain why you end up on 20

Reply:
Done, thank you suggesting this 🙇